product_price_category/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+======================
+Product Price Category
+======================
+
+This module adds a field Price Category on Product Template
+and allow Pricelist to be applied on this field.
+
+Usage
+=====
+
+In Pricelist Form (Sales -> Configuration -> Pricelist), you can add or modify
```

Comment:
Since we need `sale_management` module to configure the items, shouldn't we depend on it in the manifest?

Reply:
No, but if we depends on menus defined there